__test__/screens/ViewDetails/ViewEventScreen/ViewEventScreen.test.tsx

```diff
@@ -14,6 +14,16 @@ jest.mock("firebase/auth", () => ({
   getAuth: jest.fn(() => ({currentUser: {uid: "123"}}))
 }))
 
+jest.mock("../../../../firebase/User", () => ({
+  getUserData: jest.fn().mockResolvedValue({uid: "123"}),
+  updateUserEvents: jest.fn().mockResolvedValue(true)
+}))
+
+jest.mock("../../../../firebase/ManageEvents", () => ({
+  getEventData: jest.fn().mockResolvedValue({uid: "123", title: "Event", location: "location", point: {x: 40.712776, y: -74.005974}, date: new Date(), description: "description", imageUrl: "imageUrl"}),
```

Comment:
Maybe put the argument in a separate const for the better adaptability and readability